### DISCUSSION TITLE
EditorForm 编辑框的问题

### DISCUSSION THREAD

**Discussion by youthalan** [dotnetcore/BootstrapBlazor#4148] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

编写正常的编辑框代码
`<EditorForm class="p-2" AutoGenerateAllItem="false" RowType=RowType.Inline ItemsPerRow=2 LabelWidth=200 Model="Model">

    <FieldItems>

        <EditorItem TValue="string" TModel="L1ShutDownEntity" @bind-Field="@context.PROD_SHIFT_NO">
            <EditTemplate Context="value">
                <div class="col-12">
                    <h6>@Localizer["BasicInformation"]</h6>
                </div>
            </EditTemplate>
        </EditorItem>

        <EditorItem @bind-Field="@context.PROD_SHIFT_NO" />
        <EditorItem @bind-Field="@context.PROD_SHIFT_GROUP" />
        <EditorItem @bind-Field="@context.STOP_AREA" />
        <EditorItem @bind-Field="@context.ELAST_TIME" />
        <EditorItem @bind-Field="@context.STOP_REASON" />

    </FieldItems>

</EditorForm>`

调用的页面写上：
`<EditTemplate Context="context" >
    <ShutDownEditComponent Model="context"/>
</EditTemplate>`

如果调用页面的后台没有定义以下代码：
`    [Inject]
    [NotNull]
    private IDispatchService<bool>? DispatchService { get; set; }//用于编辑框保存时编辑按钮刷新`

`public void Dispose()
{
    DispatchService.UnSubscribe(Notify);
}

protected override Task OnInitializedAsync()
{
    DispatchService.Subscribe(Notify);
    return base.OnInitializedAsync();
}

private async Task Notify(DispatchEntry<bool> entry)
{
    //用于编辑框保存时编辑按钮刷新
    await InvokeAsync(table.QueryAsync);
    await InvokeAsync(StateHasChanged);
}
`

编辑按钮将不正常，表现为，当点编辑弹窗后，再点保存时，会让工具栏的编辑按钮状态不正常
这个应该在文档里面写清楚。

### .NET Version

8.0.400

**Comment by bb-auto** [comment]
@youthalan Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply]
@youthalan 无法复现
